roles/guest/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- name: Install linux kernel build dependencies
+  package:
+    name:
+      - libelf-dev
+      - flex
+      - bison
+    state: present
+  become: yes
+
+- name: Clone linux-guest repo
+  git:
+    repo: "{{ guest_url }}"
+    dest: "{{ guest_root }}"
+    version: "{{ guest_revision | default(omit) }}"
+    force: yes
+
+- name: Copy BKC TDX guest kernel config
+  copy:
+    src: "{{ playbook_dir }}/bkc/kafl/linux_kernel_tdx_guest.config"
+    dest: "{{ guest_root }}/.config"
+    remote_src: yes
+
+- name: Enable Virtio harness
```

Comment:
I think this + kernel config is something to do via snakemake or similar pipeline. then call that pipeline for a test case.

Reply:
Yes, this is our configurable target, and it shouldn't be part of the Ansible setup.
I just wanted a quick PoC to validate fuzz.sh at this point.
Will have a look at snakemake / similar tools later on.